docs/source/projects.rst

```diff
@@ -139,6 +112,33 @@ Docker container environment
   a Docker container environment in an ``MLproject`` file, see
   :ref:`mlproject-specify-environment`.
 
+.. _project-conda-environments:
+
+Conda environment (discouraged)
```

Comment:
```suggestion
Conda environment
```

Below, can we also tell users that, by using conda, the user is responsible for adhering to Anaconda's terms of use?

Reply:
Removed `(discouraged)`!